src/EnrollmentLinkSentController.js

```diff
@@ -127,18 +127,12 @@ function (Okta, CountryUtil, FormController, FormType, RouterUtil) {
     },
 
     remove: function () {
-      this.model.manageTransaction(function(transaction) {
-        if (transaction.stopEnrollFactorPoll) {
-          transaction.stopEnrollFactorPoll();
-        }
-      });
```

Comment:
We don't! Now the SDK automatically stops the poll when another action is taken on a transaction